hack/windows/Install-OVS.ps1

```diff
@@ -8,8 +8,8 @@ $ErrorActionPreference = "Stop"
 # TODO: set up HTTPS so that the archive can be downloaded securely. In the
 # meantime, we use a SHA256 hash to ensure that the downloaded archive is
 # correct.
-$OVSDownloadURL = "http://downloads.antrea.io/ovs/ovs-2.12.90-win64.zip"
-$OVSPublishedHash = 'B5278BE3EFFEE322DFB71FABE97B0E4F5C1341CE31D609FD60E8B41D20416C2D'
+$OVSDownloadURL = "http://downloads.antrea.io/ovs/ovs-2.13.1-win64.zip"
+$OVSPublishedHash = 'bc927e2de8560f2f0d8c2fccd6d69de3d7f4c2c032bf81658d082aa7583b241d'
```

Comment:
I thought `Get-FileHash` returned an hex string with capitalized letters and that we needed to do the same here, or the sanity check below would fail. Is that not the case?

Reply:
Thanks Antonin. I update the string to capitalized lettters.

It's interesting that the default compare for string in powershell is not case sensitive. So the lowercases can also pass the check.

Reply:
Good to know